db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/ScanCostFunction.java

```diff
@@ -147,6 +150,27 @@ private void addSelectivity(SelectivityHolder holder) {
         return holders;
     }
 
+    private void checkInListPredColumnStats(Predicate p) throws StandardException {
```

Comment:
a "check" function for me is either throwing an error or returning bool true/false = check succeeded / check failed. these two functions are filling the usedNoStatsColumnIds, so i would say a better name would be

checkInListPredColumnStats -> addNoStatsColsFromInListPred(usedNoStatsColumnIds)
checkUnaryAndBinaryPredColumnStats -> addNoStatsColsFromUnaryAndBinaryPred(usedNoStatsColumnIds)

Reply:
Done.